app/presenters/contact_presenter.rb

```diff
@@ -45,7 +45,8 @@ def format
   end
 
   def organisation
-    OpenStruct.new(details['organisation'])
+    # The only organisation in here is HMRC, so call `.first` on the array.
+    contact["links"].fetch("organisations", []).map { |organisation| OpenStruct.new(organisation) }.first
```

Comment:
You could use the `links` method here rather than accessing the contact hash directly, since you're defining the method above.